server/api/plugin.go

```diff
@@ -48,6 +48,10 @@ func newPluginHandler(handler *server.Handler, rd *render.Render) *pluginHandler
 // @Failure  500  {string}  string  "PD server failed to proceed the request."
 // @Router   /plugin [post]
 func (h *pluginHandler) LoadPlugin(w http.ResponseWriter, r *http.Request) {
+	if !h.GetScheduleConfig().EnableSchedulePlugin {
```

Comment:
I think it's better to be a `compile option`. otherwise, hackers may enable it.

Reply:
after `make pd-server-plugin`:
```bash
curl -X POST 'http://127.0.0.1:2379/pd/api/v1/plugin'                                                                                                                                                                
load plugin is disabled, please `make pd-server PLUGIN=1` first%

curl -X DELETE 'http://127.0.0.1:2379/pd/api/v1/plugin'                                                                                                                                                              
unload plugin is disabled, please `make pd-server PLUGIN=1` first%
```

Reply:
how about `PLUGIN=1 make pd-server`, similiar to `SWAGGER` and `DASHBOARD`